purchase_work_acceptance_evaluation/tests/test_purchase_work_acceptance_evaluation.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestPurchaseWorkAcceptanceEvaluation(TransactionCase):
+    def setUp(self):
+        super(TestPurchaseWorkAcceptanceEvaluation, self).setUp()
+        # Create Vendor
```

Comment:
Lots of comment is not needed. I think good code should explain itself.
Use comment only when it is really really necessary :)